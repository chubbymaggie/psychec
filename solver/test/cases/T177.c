float g();

void f()
{
    a[0].x = g();
}

