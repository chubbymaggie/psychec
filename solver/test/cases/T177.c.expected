/* Forward declarations */
typedef  struct TYPE_2__   TYPE_1__ ;

/* Type definitions */
struct TYPE_2__ {float x; } ;

/* Variables and functions */
 TYPE_1__* a ; 

